pkgs/development/python-modules/qubes-core-qrexec/default.nix

```
@@ -0,0 +1,140 @@
+{
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  qubes-core-vchan-xen,
+  pkg-config,
+  pandoc,
+  pyinotify,
+  python,
+  pygobject3,
+  gtk3,
+  wrapGAppsHook3
+}:
+
+let
+  version = "4.2.22";
+  src = fetchFromGitHub {
+    owner = "QubesOS";
+    repo = "qubes-core-qrexec";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-GxT2KpxIOeRF9j9Lc9SRHqM2G58juFtTd/lXoEJzIyY=";
+  };
+
+  agent = stdenv.mkDerivation {
+    inherit version src;
+    pname = "qubes-core-qrexec-agent";
+    sourceRoot = "${src.name}/agent";
+
+    nativeBuildInputs = [
+      pkg-config
+      pandoc
+    ];
+
+    buildInputs = [
+      qubes-core-vchan-xen
+      util-agent-bootstrap
+    ];
+
+    postInstall = ''
+      mv $out/usr/* $out/
+      rm -rf $out/usr
+      # Scripts: qubes-rpc-multiplexer
+      cp ../lib/* $out/bin/
```

```suggestion
      cp ../lib/* $out/libexec/
```
Very optional. If it breaks something, revert to `$out/bin`. Having executable libraries on libexec would make this more cohesive with the `xen` package.

---

It is installed to bit, it is just located in lib in source code.
I want however to split some of sbin binaries to libexec